packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/inspection/responses.ts

```diff
@@ -0,0 +1,200 @@
+import { RDD } from '@chainlink/gauntlet-terra'
+import { InspectInstruction, instructionToInspectCommand } from '../../../abstract/inspectionWrapper'
+import { CONTRACT_LIST } from '../../../../lib/contracts'
+import { CATEGORIES } from '../../../../lib/constants'
+import { LCDClient } from '@terra-money/terra.js'
+import { getLatestOCRNewTransmissionEvent, parseObservers, RoundData } from '../../../../lib/inspection'
+import { inspection, logger } from '@chainlink/gauntlet-core/dist/utils'
+
+type CommandInput = {
+  transmitters: string[]
+  description: string
+  operators: string[]
+  aggregatorOracles: any[]
```

Comment:
Be more explicit on the Input. What does the user need to provide exactly? Imagine the commandInput as a JSON the user has to provide for the command to perfectly work.

I also see some overlapping between transmitters, operators and aggregatorOracles. We could have a type that is `Oracle` that has all the oracle needed info.
If the user wants us to make the command input from RDD, that's fine, but the rest of the command (like the `inspect` function) should not be aware of RDD anymore.